pol-core/doc/core-changes.txt

```diff
@@ -1,4 +1,6 @@
 -- POL100.2.0 --
+01-04-2024 Kevin:
+    Added: Script script method `stacktrace( options? )`
```

Comment:
"Script object method"? I had to do two takes on "Script script" to understand it :D